Sources/DataPipeline/CustomerIO+Plugins.swift

```diff
@@ -57,21 +57,4 @@ public extension CustomerIO {
     func find(key: String) -> DestinationPlugin? {
         DataPipeline.shared.analytics.find(key: key)
     }
-
-    func setDebugLogsEnabled(_ enabled: Bool = true) {
```

Comment:
Removed this as I think it was initially added to help Tracking module and no longer needed with this change